### DISCUSSION TITLE
Zigbee2mqtt - problem npm

### DISCUSSION THREAD

**Discussion by iony59** [tteck/Proxmox#1072] [Help Needed]
Hello
I followed the procedure to install a mosquitto container and Zigbee2mqtt.
My zigate key is well seen on the server:
`root@zigbee2mqtt:/opt/zigbee2mqtt# ls -l /dev/serial/by-id`
`total 0`
`lrwxrwxrwx 1 root root 13 Feb 17 12:29 usb-Prolific_Technology_Inc._USB-Serial_Controller-if00-port0 -> ../../ttyUSB0`
my config yaml : 

homeassistant:true

permit_join: false

frontend:
 port: 9442

mqtt:
  base_topic: zigbee2mqtt
  server: 'mqtt://192.168.0.54:1883'
  user: MYUSER
  password: MYPASSWORD
  keepalive: 60
  reject_unauthorized: true
  version: 4

serial:
  port: /dev/serial/by-id/usb-Prolific_Technology_Inc._USB-Serial_Controller-if>

advanced:
  pan_id: GENERATE
  network_key: GENERATE
  channel: 20

an when i tape : 
`cd /opt/zigbee2mqtt && npm start
`
i have a error : 
`Segmentation fault`

my key is update to firmware 3.23
Thanks to help
Cédric

**Comment by tteck** [comment]
To properly display the indentations and other formatting in your `configuration.yaml` file, you will need to enclose it in a code block.

To create a code block in a post, you can surround the code with three backticks ``` at the beginning and end of the code. This will format the code block and make it easier to read.

**Comment by iony59** [comment]
sorry : 

```  Home Assistant integration (MQTT discovery)
homeassistant:true

# allow new devices to join
permit_join: false

frontend: true
 # port: 9442

# MQTT settings
mqtt:
  # MQTT base topic for zigbee2mqtt MQTT messages
  base_topic: zigbee2mqtt
  # MQTT server URL
  server: 'mqtt://192.168.0.54:1883'
  # MQTT server authentication, uncomment if required:
user: MYUSER
  password: MYPASSWORD
  keepalive: 60
  reject_unauthorized: true
  version: 4

# Serial settings
serial:
  port: /dev/serial/by-id/usb-Prolific_Technology_Inc._USB-Serial_Controller-if>
  #adapter: deconz              #(uncomment for ConBee II)
advanced:
  pan_id: GENERATE
  network_key: GENERATE
  channel: 20

**Reply by tteck** [reply]
The indentation is incorrect for `user`

**Reply by tteck** [reply]
> my key is update to firmware 3.23

What firmware are you using? Which Zigbee coordinator is this?

**Reply by iony59** [reply]
> The indentation is incorrect for `user`

but correct to yaml,

**Reply by iony59** [reply]
> > my key is update to firmware 3.23
> 
> What firmware are you using? Which Zigbee coordinator is this?

the file is "ZiGate_v3.23-OPDM.bin"

I have to look at the coordinator because I'm not near.
Zigbee2mqtt might not start because of key?

**Reply by tteck** [reply]
May not work, **alpha stage support**
https://www.zigbee2mqtt.io/guide/adapters/#based-on-zigate

**Reply by iony59** [reply]
> May not work, **alpha stage support** https://www.zigbee2mqtt.io/guide/adapters/#based-on-zigate

my zigate is USB to TTL, i try :
adapter : zigate 
but i have the same error message : segmentation fault

**Reply by iony59** [reply]
> > my key is update to firmware 3.23
> 
> What firmware are you using? Which Zigbee coordinator is this?

This is the PL-2303HX – Prolific Technology Inc.

**Comment by iony59** [comment]
so I have to go back to HA with ZHA or else buy another key?
